spec/helpers/utils.ts

```diff
@@ -22,3 +28,88 @@ export function expectError<T>(
     () => fail('expected error')
   );
 }
+
+/**
+ * Asserts that the `actual` path is equal to the `expected` one, accounting for
+ * OS differences.
+ */
+export function expectEqualPaths(actual: string, expected: string) {
```

Comment:
This mirrors the behavior I saw from dart-sass. Does dart-sass lowercase all filenames on Windows?

Reply:
Yes, the canonicalization process works around https://github.com/dart-lang/sdk/issues/35370 by eagerly converting all path names to lowercase to avoid case-sensitivity issues.